tensorflow/python/framework/config.py

```diff
@@ -237,7 +237,7 @@ def set_optimizer_experimental_options(options):
       - disable_meta_optimizer: Disable the entire meta optimizer.
       - min_graph_nodes: The minimum number of nodes in a graph to optimizer.
         For smaller graphs, optimization is skipped.
-      - Autoparallel optimizer - Automatically parallelizes graphs by splitting along the batch dimension
+      - auto_parallel : Automatically parallelizes graphs by splitting along the batch dimension
```

Comment:
Remove the space after auto_parallel for consistency.  Split across two lines, see min_graph_nodes as an example.  Max line length is 80 characters and this current change fails the linter.

Reply:
Changes done as required.Thank you!